<launch>
   <arg name="animation" default="~proto-demo" />
   <rosparam param="animation" subst_value="True">$(arg animation)</rosparam>
   <node name="teleop_demo" pkg="ros_arduino_python" type="teleop_demo.py" output="screen">
      <rosparam file="$(find ros_arduino_python)/config/arm_poses.yaml" command="load" />
   </node>
</launch>
